lib/src/repository/card_list_repo.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import 'package:hearthstoneapp/src/model/hearthstone_card.dart';
```

Comment:
This could be a relative import

Reply:
I can't use relative imports for some unknown reason, Once we discover how i'll fix them.